types/jquery.fancytree/index.d.ts

```diff
@@ -232,6 +232,8 @@ declare namespace Fancytree {
         extraClasses: string;
         /** Folder nodes have different default icons and click behavior. Note: Also non-folders may have children. */
         folder: boolean;
+        /** Icon of the tree node. */
+        icon: string;
```

Comment:
Is there something wrong with the indentation here?

Reply:
Hmm. What exactly do you mean? I don´t see any changes in the indentation in these lines.

Reply:
@Serraniel 
Sorry, I was using the Github app for Android. But I don't see any changes to the indentation on my PC.

Reply:
Sure, No Problem ;)